test/src/net.rs

```diff
@@ -1,111 +1,91 @@
-use crate::specs::TestProtocol;
-use crate::utils::{temp_path, wait_until};
-use crate::{Node, Setup};
+use crate::global::VENDOR_PATH;
+use crate::utils::{find_available_port, temp_path, wait_until};
+use crate::Node;
 use ckb_app_config::NetworkConfig;
+use ckb_chain_spec::consensus::Consensus;
+
 use ckb_channel::{self as channel, Receiver, RecvTimeoutError, Sender};
 use ckb_network::{
     bytes::Bytes, CKBProtocol, CKBProtocolContext, CKBProtocolHandler, DefaultExitHandler,
-    NetworkController, NetworkService, NetworkState, PeerIndex, ProtocolId,
+    NetworkController, NetworkService, NetworkState, PeerIndex, ProtocolId, SupportProtocols,
 };
-use ckb_types::core::{BlockNumber, BlockView};
-use std::collections::HashSet;
+
 use std::path::PathBuf;
-use std::sync::{
-    atomic::{AtomicU16, Ordering},
-    Arc,
-};
+use std::sync::Arc;
 use std::time::Duration;
 
 pub type NetMessage = (PeerIndex, ProtocolId, Bytes);
 
 pub struct Net {
-    pub nodes: Vec<Node>,
-    controller: Option<(NetworkController, Receiver<NetMessage>)>,
+    consensus: Consensus,
+    protocols: Vec<SupportProtocols>,
     p2p_port: u16,
-    setup: Setup,
     working_dir: String,
```

Comment:
please rebase with develop branch since we refactored it to `PathBuf` in https://github.com/nervosnetwork/ckb/pull/2278

Reply:
Updated.

I have resolved conflicts and pushed 2 commits: 
  * https://github.com/nervosnetwork/ckb/pull/2286/commits/6855343ccf592fbd573c7da4f49cb8c036f4b3af changes all paths into `PathBuf`

  * https://github.com/nervosnetwork/ckb/pull/2286/commits/dc97f66f239a551fe449cfe4ebbc15540cc9ef82 remove `log_size` out from `Node`. I think `log_size` is a rarely-use function, it is unnecessary to place inside `Node`.

cc @chuijiaolianying